src/events/external_clients.py

```diff
@@ -63,18 +62,15 @@ def _on_mtls_cert_updated(self, event: MTLSCertUpdatedEvent) -> None:  # noqa: C
         """Handle the ca chain updated event."""
         if not event.mtls_cert or not event.prefix:
             logger.error("CA chain, keys prefix, or common name not provided")
-            self.charm.set_status(Status.EC_MISSING_CREDENTIALS)
```

Comment:
I'm not clear why these statuses are no longer shown? Are they offloaded to the TLS module? in which case how are the ones like `EC_USERNAME_EXISTS` handled?